docs/deployments/azure/running-azure-powershell/configuring-the-version-of-the-azure-powershell-modules.md

```diff
@@ -3,7 +3,13 @@ title: Configuring the version of the Azure PowerShell Modules
 description: A version of Azure PowerShell is bundled with Octopus Deploy and it's possible to configure which version you wish to use in your deployments.
 ---
 
-A version of Azure PowerShell is bundled with Octopus Deploy.  To determine the versions of the various Azure modules, add the PowerShell below to an Azure PowerShell Script Step:
+:::warning
+Using the Azure tools bundled with Octopus Deploy is not recommended. Octopus bundles versions of the Azure Resource Manager Powershell modules (AzureRM) and Azure CLI. These were originally provided as convenience mechanisms for users wanting to run scripts against Azure targets. The versions bundled are now out of date, and we will not be updating them further.
+:::
+
+We recommend you configure Octopus Deploy to use your own [version of the Azure PowerShell cmdlets](configuring-the-version-of-the-azure-powershell-modules.md).
```

Comment:
Same question as above. Link goes to the page the reader is already on?

Reply:
Good spotting, I've removed those.